### PR TITLE
added remove email function

### PR DESCRIPTION
**Pull request by AaronBasch**
see title

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -43,12 +43,14 @@ def main():
 							print('3')
 						elif account_choice == 4:
 							email = input("Enter email to remove: ")
-							
+							print(queries.add_new_email(email))
```
Comment (vkoli) [inline main.py:46]: This should call queries.remove_email() not add_new_email(). Otherwise get a runtime error.

File: queries.py [hunk 2]
```diff
@@ -26,11 +26,19 @@ def add_new_email(email,user_ssn,verified=1):
...
 
-def remove_email():
-    pass
-
-def add_new_phone():
-    pass
+def remove_email(email):
+    return connect.exec(f"""DELETE FROM ELEC_ADDRESS WHERE Identifier='{email}');\n
+                            DELETE FROM EMAIL WHERE Identifier='{email}');\n""")
+
```
Comment (vkoli) [inline queries.py:41]: - This also needs to delete the rows in SEND_TRANSACTION and REQUEST_TRANSACTION that reference the deleted email.
- Minor syntax error: you have a single closed parantheses ) still at the end of line 30 and line 31

File: queries.py [hunk 3]
```diff
@@ -26,11 +26,19 @@ def add_new_email(email,user_ssn,verified=1):
...
+def update_phone(phone,user_ssn):
+    return connect.exec(f"""UPDATE ELEC_ADDRESS
+                            SET Identifier='{phone}'
+                            WHERE Identifier=   (SELECT PhoneNo
+                                                FROM USER_ACCOUNT
+                                                WHERE SSN='{user_ssn}');
+                            UPDATE USER_ACCOUNT
+                            SET PhoneNo='{phone}'
+                            WHERE SSN='{user_ssn}';""")
```
Comment (vkoli) [inline queries.py:61]: Same thing here. This needs to also update any identifier in SEND_TRANSACTION AND REQUEST_TRANSACTION that referenced the old phone.

**Review by vkoli — CHANGES_REQUESTED** [review]
Everythin

**Comment by AaronBasch** [comment]
Right! Yes will fix these issues today. Thanks!

**Pull request merged**